### PR TITLE
chore: modify-license

### PR DESCRIPTION
**Pull request by jrwbabylonlab**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -8,12 +8,82 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor:             Babylon Labs, Ltd.
+Licensor:             Babylon Labs Ltd.
 
 Licensed Work:        staking-api-service
```
Comment (gitferry) [inline LICENSE:13]:
```suggestion
Licensed Work:        Staking API Service
```

File: LICENSE [hunk 2]
```diff
@@ -8,12 +8,82 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
...
+8.  in violation of applicable laws, rules or regulations in your relevant jurisdiction;
+9.  that circumvents any sanctions or export controls targeting you or the country or territory where
+    you are located; or
+10. in any activity that transmits, exchanges, or is otherwise supported by the direct or indirect
+    proceeds of criminal or fraudulent activity.
+
+Any production use of the Licensed Work by you confirms your agreement to the foregoing limitations,
+and your understanding and agreement that they are limitations and not restrictions.
 
 Change Date:          2027-05-15 (March 15th, 2027)
```
Comment (gitferry) [inline LICENSE:88]:
```suggestion
Change Date:          2027-03-15 (March 15th, 2027)
```

**Review by gitferry — APPROVED** [review]

**Review by jeremy-babylonlabs — APPROVED** [review]

File: LICENSE [hunk 3]
```diff
@@ -8,14 +8,84 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor:             Babylon Labs, Ltd.
+Licensor:             Babylon Labs Ltd.
 
-Licensed Work:        staking-api-service
+Licensed Work:        Staking API Service
```
Comment (vitsalis) [inline LICENSE:13]: seems it has additional whitespace at the end

File: LICENSE [hunk 4]
```diff
@@ -8,14 +8,84 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor:             Babylon Labs, Ltd.
+Licensor:             Babylon Labs Ltd.
 
-Licensed Work:        staking-api-service
+Licensed Work:        Staking API Service  
                       The Licensed Work is (c) 2024 Babylon Labs, Ltd.
```
Comment (vitsalis) [inline LICENSE:14]:
```suggestion
                      The Licensed Work is (c) 2024 Babylon Labs Ltd.
```

**Review by vitsalis — APPROVED** [review]

**Pull request merged**